x-pack/test/security_solution_cypress/cypress/e2e/explore/dashboards/upgrade_risk_score.cy.ts

```diff
@@ -44,152 +41,69 @@ describe('Upgrade risk scores', { tags: ['@ess', '@serverless', '@brokenInServer
   before(() => {
     cleanKibana();
     login();
+    deleteRiskEngineConfiguration();
     createRule(getNewRule({ rule_id: 'rule1' }));
   });
 
-  beforeEach(() => {
-    login();
-    deleteRiskScore({ riskScoreEntity: RiskScoreEntity.host, spaceId });
-    deleteRiskScore({ riskScoreEntity: RiskScoreEntity.user, spaceId });
-    installLegacyRiskScoreModule(RiskScoreEntity.host, spaceId);
-    installLegacyRiskScoreModule(RiskScoreEntity.user, spaceId);
-    visitWithTimeRange(ENTITY_ANALYTICS_URL);
-  });
+  describe('show upgrade risk button', () => {
+    beforeEach(() => {
+      login();
+      deleteRiskScore({ riskScoreEntity: RiskScoreEntity.host, spaceId });
+      deleteRiskScore({ riskScoreEntity: RiskScoreEntity.user, spaceId });
+      installLegacyRiskScoreModule(RiskScoreEntity.host, spaceId);
+      installLegacyRiskScoreModule(RiskScoreEntity.user, spaceId);
+      visitWithTimeRange(ENTITY_ANALYTICS_URL);
+    });
 
-  it('shows upgrade risk button for host and user', () => {
-    cy.get(UPGRADE_HOST_RISK_SCORE_BUTTON).should('be.visible');
-    cy.get(UPGRADE_USER_RISK_SCORE_BUTTON).should('be.visible');
-  });
+    afterEach(() => {
+      deleteRiskScore({ riskScoreEntity: RiskScoreEntity.host, spaceId });
```

Comment:
As we are doing the deletion on the beforeEach hook, we might not need it on the before each.